packages/flame/lib/src/parallax.dart

```diff
@@ -234,6 +343,63 @@ class ParallaxLayer {
 /// How to fill the screen with the image, always proportionally scaled.
 enum LayerFill { height, width, none }
 
+abstract class ParallaxData {
```

Comment:
Maybe some dartdocs as to what use this class has for those who want to implement it?